### ISSUE TITLE
Alphabetically sort category list

### ISSUE THREAD

**Issue by jpds** [organicmaps/organicmaps#2188]
The category list has recently grown, and I'm not sure what is used to sort the items - perhaps it should be alphabetically sorted so people can locate things easier?

**Comment by biodranik** [comment]
Yes, we plan to resort it alphabetically, with the `en` language going first. Can you help to resort it? (after merging all existing PRs).

**Comment by Charlie-1-3** [comment]
i would like to take up this issue

**Comment by pastk** [comment]
> i would like to take up this issue

Please check the existing script `tools/python/categories_converter.py`

Its description says

> A script for converting categories.txt into the strings.txt format and back, as well as for autoformatting categories.txt.

Perhaps it needs to be fixed/updated.
